Objects/abstract.c

```diff
@@ -168,6 +169,25 @@ PyObject_GetItem(PyObject *o, PyObject *key)
                               "be integer, not '%.200s'", key);
     }
 
+    if (PyType_Check(o)) {
+        meth = PyObject_GetAttrString(o, "__class_getitem__");
```

Comment:
This should probably use a `Py_IDENTIFIER`.

Reply:
Yes, when making an actual PR to ``python/cpython`` I will probably switch to ``_PyObject_GetAttrId``.